src/app/Library/CrudPanel/Traits/Search.php

```diff
@@ -166,6 +166,20 @@ public function getPersistentTable()
         return config('backpack.crud.operations.list.persistentTable');
     }
 
+    /**
+     * Get duration for persistent table.
+     *
+     * @return bool
+     */
+    public function getPersistentTableDuration()
```

Comment:
In the future, I would refrain from adding more getters & setters, when they only return an operation setting. I think we have enough of that already :-))

My thinking is - the more people use the default syntax for the Settings API, the more they're likely to code their own settings and features, and store and retrieve settings using the Settings API. Let me know what you think.

Plus, if setting calls would all use the Settings API syntax (instead of each having their own method name), the calls would look pretty clean next to each other, and they would be self-explanatory.

Just my 2 cents.

Reply:
Could not agree more. I will leave this in my notes so in a spare time I refactor persistent table to use Settings API :)

In future I will take this in consideration :+1: